sigmastate/src/main/scala/sigmastate/eval/CostingDataContext.scala

```diff
@@ -633,8 +631,12 @@ class CostingSigmaDslBuilder extends TestSigmaDslBuilder { dsl =>
                                  positions: Coll[Int],
                                  newValues: Coll[T])
                                 (implicit cT: RType[T]): Coll[Byte] = {
-    val typedNewVals = newValues.toArray.map(_.asInstanceOf[Value[SType]])
-    val res = SubstConstants.eval(scriptBytes.toArray, positions.toArray, typedNewVals)
+    val typedNewVals = newValues.toArray.map(v => TransformingSigmaBuilder.liftAny(v) match {
```

Comment:
ScalaDoc missed

Reply:
added ScalaDocs for all methods of SigmaDslBuilder, and all public methods of CostingDataContext which are not marked `override`